overthrower/overthrower.cpp

```diff
@@ -272,7 +283,8 @@ void* malloc(size_t size)
 #if !defined(__APPLE__)
     if (activated != 0 && pointer != NULL) {
         std::lock_guard<std::mutex> lock(mutex);
-        allocated.insert(pointer);
+        alloc_counter++;
+        allocated.insert({pointer, alloc_counter});
```

Comment:
Potentially `emplace` can be more efficient than `insert` and should be never be less efficient.